action/library/src/main/java/io/knotx/fragments/action/library/InMemoryCacheActionFactory.java

```diff
@@ -67,173 +44,17 @@
 @Cacheable
 public class InMemoryCacheActionFactory implements ActionFactory {
```

Comment:
Do we still need this action?
Why not just a generic `CacheActioinFactory`?

Reply:
We would lose backwards compatibility (`InMemoryCacheActionFactory` is registered with `in-memory-cache` name, which would stop working if we removed this class).
Not sure if it's a big deal, but I recall that breaking changes like this should increase the major release version?

Reply:
I agree, we can mark it as deprecated.